### PR TITLE
[CI:DOCS] Migrate skopeo container image

### PR DESCRIPTION
**Pull request by cevich**
Moved to: https://github.com/containers/image_build

This is needed to consolidate both the content and the automation into one location. Having both in the same place is intended to both protect push secrets and allow for PR-level CI testing.

### CODE REVIEW

File: contrib/skopeoimage/README.md [hunk 1]
```diff
@@ -1,68 +1,2 @@
...
-
-# Copy the Skopeo container image from quay.io to
-# a private registry
-podman run docker://quay.io/skopeo/stable:latest copy docker://quay.io/skopeo/stable docker://registry.internal.company.com/skopeo
-
-# Inspect the fedora:latest image
-podman run docker://quay.io/skopeo/stable:latest inspect --config docker://registry.fedoraproject.org/fedora:latest  | jq
-```
+The skopeo container image build context and automation have been
+moved to [https://github.com/containers/image_build](https://github.com/containers/image_build)
```
Comment (edsantiago) [inline contrib/skopeoimage/README.md:2]: LGTM but it might've been friendlier to link directly into the `/skopeo` subdirectory

**Review by edsantiago — APPROVED** [review]

**Review by edsantiago — APPROVED** [review]

File: contrib/skopeoimage/README.md (hunk 1 above)
Comment (cevich) [reply contrib/skopeoimage/README.md:2]: That's not a bad idea, I can do that.

**Review by mtrmac — COMMENTED** [review]
ACK.

`install.md` is pointing at `./contrib/skopeoimage/README.md`, changing that link to point directly to the new location would save readers a click.

**Comment by TomSweeneyRedHat** [comment]
LGTM

**Comment by cevich** [comment]
@mtrmac good suggestion.  I just force-pushed with a `install.md` change as well.

**Review by mtrmac — APPROVED** [review]

**Comment by mtrmac** [comment]
Please merge whenever appropriate.

**Pull request merged**